### PR TITLE
add: content/pt-br/serverless.md

### PR DESCRIPTION
**Pull request by luizgribeiro**
Add

- content/pt-br/serverless.md

Please, provide feedback for any term and/or word that is not an adequate translation of the original one.

### CODE REVIEW

File: content/pt-br/serverless.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+Title: Serverless
+Status: Completed
+Category: Technology
+---
+
+## O que é
+
+Serverless é um modelo de desenvolvimento nativo na nuvem que permite que desenvolvedores criem aplicações sem que seja necessário gerenciar servidores. Ainda existem servidores no modelo serverless, mas eles são [abstraídos](/abstraction) do desenvolvimento. Um provedor de computação em nuvem se encarrega do provisionamento, manutenção e [escala](/scalability/) da infraestrutura de servidores. Desenvolvedores podem simplesmente carregar o código em [containers](/container/) para implantação. Uma vez implantadas, aplicações serverless respondem automaticamente à demanda, escalando para cima e para baixo conforme necessário. O preço cobrado por provedores de computação em nuvem para funções serverless geralmente é realizada com base na sua utilização sob demanda. Como resultado disto, quando uma função serverless não está em execução ela não tem custo algum.
```
Comment (jihoon-seo) [inline content/pt-br/serverless.md:9]: - nit: adding trailing slash to hyperlinks will remove unnecessary redirects
  (`https://glossary.cncf.io/abstraction` → `https://glossary.cncf.io/abstraction/`).
- There exists a `pt-br` version of "Containers" page.

```suggestion
Serverless é um modelo de desenvolvimento nativo na nuvem que permite que desenvolvedores criem aplicações sem que seja necessário gerenciar servidores. Ainda existem servidores no modelo serverless, mas eles são [abstraídos](/abstraction/) do desenvolvimento. Um provedor de computação em nuvem se encarrega do provisionamento, manutenção e [escala](/scalability/) da infraestrutura de servidores. Desenvolvedores podem simplesmente carregar o código em [contêineres](/pt-br/container/) para implantação. Uma vez implantadas, aplicações serverless respondem automaticamente à demanda, escalando para cima e para baixo conforme necessário. O preço cobrado por provedores de computação em nuvem para funções serverless geralmente é realizada com base na sua utilização sob demanda. Como resultado disto, quando uma função serverless não está em execução ela não tem custo algum.
```

File: content/pt-br/serverless.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Problema relacionado
+
+Em um modelo de [computação em nuvem](/cloud_computing/) com [Infraestrutura-como-Serviço (IaaS)](/infrastructure_as_a_service), usuários adquirem unidades de capacidade, o que significa que o provedor da nuvem fornece componentes de servidores sempre disponíveis para a execução de aplicações. É responsabilidade do usuário fazer com que os servidores escalem para cima durante períodos de alta demanda e que escalem para baixo quando a capacidade fornecida não é mais necessária. A infraestrutura de nuvem utilizada para executar uma aplicação fica, no entanto, ativa mesmo quando a aplicação não está sendo utilizada.
```
Comment (jihoon-seo) [inline content/pt-br/serverless.md:13]: nit: adding trailing slash to hyperlinks will remove unnecessary redirects
(`https://glossary.cncf.io/infrastructure_as_a_service` → `https://glossary.cncf.io/infrastructure_as_a_service/`).

```suggestion
Em um modelo de [computação em nuvem](/cloud_computing/) com [Infraestrutura-como-Serviço (IaaS)](/infrastructure_as_a_service/), usuários adquirem unidades de capacidade, o que significa que o provedor da nuvem fornece componentes de servidores sempre disponíveis para a execução de aplicações. É responsabilidade do usuário fazer com que os servidores escalem para cima durante períodos de alta demanda e que escalem para baixo quando a capacidade fornecida não é mais necessária. A infraestrutura de nuvem utilizada para executar uma aplicação fica, no entanto, ativa mesmo quando a aplicação não está sendo utilizada.
```

File: content/pt-br/serverless.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+Title: Serverless
+Status: Completed
+Category: Technology
```
Comment (jihoon-seo) [inline content/pt-br/serverless.md:4]: Other `pt-br` terms are using the category "tecnologia".

- https://github.com/cncf/glossary/blob/main/content/pt-br/container.md?plain=1#L4
- https://github.com/cncf/glossary/blob/main/content/pt-br/containerization.md?plain=1#L4
- ...

```suggestion
Category: tecnologia
```

**Review by jihoon-seo — COMMENTED** [review]
@luizgribeiro PTAL!

**Review by luizgribeiro — COMMENTED** [review]
Thanks for the review @jihoon-seo!

I'll pay more attention to those points that you highlighted on future translations.

Cheers!

File: content/pt-br/serverless.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+Title: Serverless
+Status: Completed
+Category: tecnologia
+---
+
+## O que é
+
+Serverless é um modelo de desenvolvimento nativo na nuvem que permite que desenvolvedores criem aplicações sem que seja necessário gerenciar servidores. Ainda existem servidores no modelo serverless, mas eles são [abstraídos](/abstraction/) do desenvolvimento. Um provedor de computação em nuvem se encarrega do provisionamento, manutenção e [escala](/scalability/) da infraestrutura de servidores. Desenvolvedores podem simplesmente carregar o código em [contêineres](/pt-br/container/) para implantação. Uma vez implantadas, aplicações serverless respondem automaticamente à demanda, escalando para cima e para baixo conforme necessário. O preço cobrado por provedores de computação em nuvem para funções serverless geralmente é realizada com base na sua utilização sob demanda. Como resultado disto, quando uma função serverless não está em execução ela não tem custo algum.
```
Comment (edsoncelio) [inline content/pt-br/serverless.md:9]: We already have `/abstraction` in Portuguese so you can use `pt-bt/abstraction`.

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Hi @luizgribeiro! first of all, thanks for your contribution! I left just a small request, after that it's a LGTM :)

**Review by luizgribeiro — COMMENTED** [review]
Thanks @edsoncelio, I've just added the path.

I was also wondering if it could be possible to enforce the usage of the language directory (in our case pt-br) and, if the article does not exist redirect it to the english/original version

**Review by edsoncelio — APPROVED** [review]

**Pull request merged**